### ISSUE TITLE
Very good softwarre

### ISSUE THREAD

**Issue by jumpingjackflash5** [FlyGoat/RyzenAdj#141]
Thank your for this program I am using it with Ryzen 3200U.
I use simple scheduling - command line - with reapply after wakeup from S3 and after bootup. Works fine for keeping the parameters set.

**Comment by Falcosc** [comment]
I did need to add Power Source Change event to my task scheduler. But I got timing issues at some point and switch to a loop. (stock profile from Power Source change got applied after ryzenAdj and overwrite it)

**Comment by jumpingjackflash5** [comment]
Thank you for answer. I use just event after S3 wakeup (and startup) and then run ryzenadj.exe with correct parameters.

![obrazek](https://user-images.githubusercontent.com/49901997/114870556-990abc80-9df8-11eb-9c07-a8c010ebacef.png)

**Comment by Falcosc** [comment]
If you want to add power source change, you need to prove a XML configuration for the query if I remember correctly.

I did a quick look, you can also add the power source into the trigger to have 2 actions, one for battery and one for AC

```
    <QueryList> 
      <Query Id="0" Path="System"> 
        <Select Path="System"> 
           *[System[Provider[@Name='Microsoft-Windows-Kernel-Power'] and EventID=105] and EventData[Data[@Name='AcOnline']='false'] ]    
        </Select> 
      </Query> 
    </QueryList> 
```

**Comment by jumpingjackflash5** [comment]
Yes it may be done this way. But for me it enough to reapply one set of settings.
